modin/config/envvars.py

```diff
@@ -34,7 +34,7 @@
 )
 
 
-class EnvironmentVariable(Parameter, type=str, abstract=True):
+class EnvironmentVariable(Parameter, type=str, abstract=True):  # noqa: PR01
```

Comment:
I've added `__init__(self, value)` to the base class, and apparently, all the derived classes also require a doc-string for the `value` parameter, added this noqa instead